internal/commands/accesslist/output.go

```diff
@@ -0,0 +1,38 @@
+package accesslist
+
+import "github.com/10gen/realm-cli/internal/cloud/realm"
+
+const (
+	headerAddress = "IP Address"
+	headerComment = "Comment"
+)
+
+var (
+	tableHeaders = []string{headerAddress, headerComment}
+)
+
+type allowedIPOutputs []allowedIPOutput
```

Comment:
this looks like some copy/paste that ultimately was never used

fwiw, i don't think having this abstraction is necessary yet until we're printing allowed ips in tabular format in more than one place.  seems like you could move `tableHeaders` to the `list.go` file (maybe rename it to `listTableHeaders`) and then delete this whole file